back/engines/commercial/verification/app/services/verification/patches/permissions/user_requirements_service.rb

```diff
@@ -17,20 +17,20 @@ def requires_verification?(permission, user)
         def base_requirements(permission)
           requirements = super
 
+          # TODO: JS - does check groups need to only work with group based verification?
           if @check_groups && permission.verification_enabled?
-            requirements[:special][:verification] = 'require'
+            requirements[:verification] = true
           end
           requirements
         end
 
         def mark_satisfied_requirements!(requirements, permission, user)
           super
-          return unless requirements[:special][:verification] == 'require'
+          return unless requirements[:verification]
 
-          if user.verified?
-            requirements[:special][:verification] = 'satisfied'
-          elsif user_allowed_through_other_groups?(permission, user)
-            requirements[:special][:verification] = 'dont_ask'
+          # TODO: JS - does the front-end care about the difference between dont_ask and satisfied that we had before?
```

Comment:
No, we only check if the value is `'required'` or not. So seems good to simplify to a boolean.